.github/workflows/ci.yml

```diff
@@ -57,3 +65,7 @@ jobs:
       run: COMMAND=./dist/kubectx_linux_amd64_v1/kubectx bats test/kubectx.bats
     - name: kubens (Go) integration tests
       run: COMMAND=./dist/kubens_linux_amd64_v1/kubens bats test/kubens.bats
+    - name: Publish Snaps to the Snap Store (edge channel)
```

Comment:
Why is this step in "ci"? Shouldn't it be release-only?

Reply:
It's to publish a SNAPSHOT release on edge channel with each new pipeline run. It's very useful to debug things, but I can remove and just keep the main run on release.yaml. Your call.

Reply:
![image](https://github.com/ahmetb/kubectx/assets/6025031/5f736c85-e632-4910-aaf7-921e19049dad)

Reply:
But this will run even on unmerged PR tests. I don't think that's safe or trusted.

Reply:
Ok then. I will remove. Give 1 min.

Reply:
Done. Now it will just run with release.yaml. Any other considerations?